bigtop-packages/src/common/sqoop/patch4-upgrade-ivy-250.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/ivy/libraries.properties b/ivy/libraries.properties
+index 2ef04f4f..11f17c36 100644
+--- a/ivy/libraries.properties
++++ b/ivy/libraries.properties
+@@ -34,7 +34,7 @@ commons-net.version=3.1
+ 
+ hsqldb.version=1.8.0.10
+ 
+-ivy.version=2.3.0
++ivy.version=2.5.0
```

Comment:
Would you include this change into patch3 directly, since this PR is addendum to that?